### PR TITLE
DIV-4495 Workflow position updated for ContactDetails page

### PR DESCRIPTION
**Pull request by Dizrupto**
# Description

Modified the position of the petitioner contact details page to be earlier in the workflow.

Solves https://tools.hmcts.net/jira/browse/DIV-4495

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

This story only required modification of existing tests. No additional tests were required.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/petitioner/changed-name/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = class PetitionerChangedNamed extends ValidationStep {
     return '/petitioner-respondent/changed-name';
   }
   get nextStep() {
-    return this.steps.PetitionerContactDetails;
+    return this.steps.PetitionerHomeAddress;
```
Comment (jvictorcf) [inline app/steps/petitioner/changed-name/index.js]: What's this page?

Comment (Dizrupto) [reply app/steps/petitioner/changed-name/index.js]: That's the next page in the flow. If I remember well, this is where the petitioner enters their post code and it finds the address

Comment (Dizrupto) [reply app/steps/petitioner/changed-name/index.js]: Before hand, the flow was: /changed-name -> /contact-details -> /home-address.
But now the contact-details page has been moved earlier in the flow so the /changed-name directly points to the /home-address

**Comment by hosainnet** [comment]
@Dizrupto  is this still needed? It's from the 15th of May

**Comment by dtaylor7** [comment]
closing due to age

**Pull request closed without merging**